src/home_robot/home_robot/mapping/voxel.py

```diff
@@ -155,15 +155,30 @@ def get_instances(self) -> List[InstanceView]:
         """Return a list of all viewable instances"""
         return tuple(self.instances.instance_views[0].values())
 
-    def add_obs(self, obs: Observations, *args, **kwargs):
+    def fix_type(self, tensor: Union[np.ndarray, torch.Tensor]) -> torch.Tensor:
+        """Convert to tensor and float"""
+        if tensor is None:
+            return None
+        if isinstance(tensor, np.ndarray):
+            tensor = torch.from_numpy(tensor)
+        return tensor.float()
+
+    def add_obs(
```

Comment:
@alexsax Do you think you could make sure this SparseVoxelMap code works with your class? I think i modified it so that it should support it from here on, we can definitely pull these changes into main via your branch